gui/default/syncthing/core/syncthingController.js

```diff
@@ -3077,6 +3077,120 @@ angular.module('syncthing.core')
                     address.indexOf('unix://') == 0 ||
                     address.indexOf('unixs://') == 0);
         };
+
+        $scope.shareDeviceConfirmationModal = function (type) {
```

Comment:
Yep, I've changed it to `type` because of the very similar function used for the revert local additions confirmation. Do you think it's better to go with `method` even though the existing and very similar code uses `type`? I was just trying to be consistent here.

Reply:
Yes.